rvv-intrinsic-rfc.md

```diff
@@ -348,70 +348,131 @@ vmerge.vvm vd, vs2, vs1, v0:
 vint8m1_t vmerge_vvm_i8m1_m(vbool8_t mask, vint8m1_t vs2, vint8m1_t vs1, size_t vl);
 ```
 
-## Tail Policy in the Intrinsics<a name="tail-policy"></a>
+## Policy Intrinsic Functions<a name="policy"></a>
 
-The naming rule of intrinsics with tail policy is
+The intrinsics encode the policy (or policies) explicitly in function name's suffix if operations support those policies.
+Users could use policy intrinsic functions to control tail and inactive elements. Please refer to [3.4.3 Vector Tail Agnostic and Vector Mask Agnostic](https://github.com/riscv/riscv-v-spec/blob/master/v-spec.adoc#343-vector-tail-agnostic-and-vector-mask-agnostic-vta-and-vma)
+and [5.4 Prestart, Active, Inactive, Body, and Tail Element Definitions](https://github.com/riscv/riscv-v-spec/blob/master/v-spec.adoc#sec-inactive-defs) for fundamental definitions.
+
+For policy intrinsic functions:
+
+- When it is a masked intrinsic...
+  - If the mask policy is not explicitly specified, the policy will depend on whether `vundefined()` is passed to the `merge` argument
+  - If the tail policy is not explicitly specified, the policy is set to "agnostic"
+
+- When it is an unmasked intrinsic...
+  - If the tail policy is not explicitly specified, the policy is set to "agnostic".
+
+The general naming rule is as the following.
+The interface differs between instructions and will be enumerated in the following of this section.
 
 ```
-INTRINSIC_WITH_ADDITIONAL_DEST ::= INTRINSIC '_tu'
+INTRINSIC_WITH_TU ::= INTRINSIC '_tu'
+INTRINSIC_WITH_TA ::= INTRINSIC '_ta'
+INTRINSIC_WITH_MASK_TU ::= INTRINSIC '_m_tu'
+INTRINSIC_WITH_MASK_TA ::= INTRINSIC '_m_ta'
+INTRINSIC_WITH_MASK_MU ::= INTRINSIC '_mu'
+INTRINSIC_WITH_MASK_MA ::= INTRINSIC '_ma'
+INTRINSIC_WITH_MASK_TU_MA ::= INTRINSIC '_tuma'
+INTRINSIC_WITH_MASK_TA_MA ::= INTRINSIC '_tama'
+INTRINSIC_WITH_MASK_TU_MU ::= INTRINSIC '_tumu'
+INTRINSIC_WITH_MASK_TA_MU ::= INTRINSIC '_tamu'
+```
+
+### Intrinsics without `dest` or `vd` argument
+
+|Masked?  | TU? | MU? | Intrinsic
+|--- |--- |--- |---
+|No  | No | N/A| `vadd_vv_<ty>_ta(vs2, vs1, vl)` and `vadd_vv_i8m1(vs2, vs1, vl)`
+|No  | Yes| N/A| `vadd_vv_<ty>_tu(merge,vs2, vs1, vl)`
+|Yes | No | No | `vadd_vv_<ty>_tama(mask, vs2, vs1, vl)`
+|Yes | No | Yes| `vadd_vv_<ty>_tamu(mask, merge, vs2, vs1, vl)`
+|Yes | Yes| No | `vadd_vv_<ty>_tuma(mask, merge, vs2, vs1, vl)`
+|Yes | Yes| Yes| `vadd_vv_<ty>_tumu(mask, merge, vs2, vs1, vl)` and `vadd_vv_i8m1_m(mask, merge, vs2, vs1, vl)`
+
+### Intrinsics with the `dest` argument and doesn't have a masked type
```

Comment:
Missing a section for "Intrinsics with the `dest` argument and **have** a masked type".

Reply:
In fact, Intrinsics with the `vd` argument is the same as "Intrinsics with the dest argument and have a masked type" because we call it `vd` rather than `dest`. https://github.com/riscv-non-isa/rvv-intrinsic-doc/pull/104